moviepy/config.py

```diff
@@ -56,7 +56,11 @@ def try_cmd(cmd):
             IMAGEMAGICK_BINARY = wr.QueryValueEx(key, 'BinPath')[0] + r"\convert.exe"
             key.Close()
         except:
-            IMAGEMAGICK_BINARY = 'unset'
+            try:
+                imagemagick_path   = sp.check_output('dir /B /O-N "C:\Program Files\ImageMagick-*"'                           , shell=True).decode("utf-8").split("\r\n")[0]
+                IMAGEMAGICK_BINARY = sp.check_output('dir /B /S   "C:\Program Files\{}\*convert.exe"'.format(imagemagick_path), shell=True).decode("utf-8").split("\r\n")[0]
+            except:
```

Comment:
Also, would be better to catch an explicit exception. A catch-all (whitout even log it) is unpythonic.

Reply:
I don't see the benefit of an explicit exception here.
If it doesn't work then you end up with the error message that imagemagick couldn't be found and you have to add the path manually (that output is generated somewhere else in moviepy).